config/app.config

```diff
@@ -1,14 +1,15 @@
 //---------------------------------------------
 // adjustable application runtime properties
 //---------------------------------------------
-BuildMajor = 2022
-BuildMinor = 6
-BuildRev = 0
+BuildMajor = 2023
+BuildMinor = 1
+BuildRev = 3
```

Comment:
Let's bump the build number for the release.

Reply:
In the long run I would prefer *not* to have the suit versions directly track the Firefly versions; they won't and should not always update together.